code/modules/projectiles/ammunition/bullets.dm

```diff
@@ -1,3 +1,9 @@
+/obj/item/ammo_casing/a556x45
+	desc = "A 5.56x45 bullet casing."
+	caliber = "556x45"

```

Comment:
Save up characters, also I don't think periods play nice, I already had trouble with DM thinking I was writing an int in the new ammo type